rtos/TARGET_CORTEX/mbed_rtx_conf.h

```diff
@@ -45,7 +45,11 @@
 #error "OS Tickrate must be 1000 for system timing"
 #endif
 
-#if !defined(OS_STACK_WATERMARK) && (defined(MBED_STACK_STATS_ENABLED) && MBED_STACK_STATS_ENABLED)
+#if !defined(OS_STACK_WATERMARK) && defined(MBED_STACK_STATS_ENABLED)
```

Comment:
Do we need it duplicated? Can we use `#if !defined(OS_STACK_WATERMARK) && (defined(MBED_ALL_STATS_ENABLED) || defined(MBED_STACK_STATS_ENABLED))`